File: challenges/2-idade-em-dias/java/izepon/IdadeEmDias.java
import java.util.Scanner;

public class IdadeEmDias {
    public static void main(String[] args) {
    	
    	Scanner sc = new Scanner(System.in);
    	
        int daysPerYear = 365;
        int daysPerMonth = 30;    	      

        System.out.printf("Digite aqui sua Idade em Dias: ");
        int age = sc.nextInt();

        int years = age / daysPerYear;
        int month = (age % daysPerYear) / daysPerMonth;
        int days = (age % daysPerYear) % daysPerMonth;
        
        System.out.println(years + " ano(s)\n" + month + " mes(es)\n" + days + " dia(s)");

        sc.close();
    }
}

File: challenges/2-idade-em-dias/php/akiratorres/idadeEmDias.php
<?php

$ano = 0;
$mes = 0;
$dia = 0;

$age = (int)readline("Digite a sua idade em dias: ");

while ($age >= 365) {
    $ano++;
    $age -= 365;   
}

while ($age >= 30) {
    $mes++;
    $age -= 30;
}

while ($age >= 1){
    $dia++;
    $age--;
}

echo "$ano ano(s)\n$mes mes(es)\n$dia dia(s)" . PHP_EOL;
